README.md

```diff
@@ -34,9 +36,41 @@ For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackr
     + [How to Deploy](#how-to-deploy)
 * [Generating portable installers](#generating-portable-installers)
 
-## Quick Installation via Helm
+## Installation via Helm
+
+### Quick Installation
 
 StackRox offers quick installation via Helm Charts. Follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) to get `helm` CLI on your system.
+Then run the helm quick install script or proceed to section [Manual Helm Installation](#manual-helm-installation) for configuration options.
+```sh
+./scripts/quick-helm-install.sh 
+```
+The script will add the stackrox helm repository, generate an admin password, install stackrox-central-services, create an init bundle for shared secrets, and finally create one secured cluster.
+If you wish to create more secured clusters you may do so using
+```sh
+helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services \
+  -f stackrox-init-bundle.yaml \
+  --set clusterName="$CLUSTER_NAME"
+```
+
+#### First use
+
+Follow these steps to get to StackRox UI
+
+1. Setup port forward to central:
+```sh
+kubectl -n stackrox port-forward deploy/central 8000:8443
+```
+
+2. Open <https://localhost:8000> in your browser.
+
+3. If a certificate warning is displayed, you can accept the warnings to install the default self-signed certificate. To configure custom certificates, see the section on "Adding Custom Certificates" in <https://docs.openshift.com/acs/configuration/add-custom-certificates.html>.
+
+4. Log in as `admin` using the password in `STACKROX_ADMIN_PASSWORD`
```

Comment:
1. If things are ambiguous and can be misunderstood, they **will** be misunderstood.
In this case, what is `STACKROX_ADMIN_PASSWORD`? Maybe you could add a command that just reveals the password for users so they can copy-paste it from the terminal?
2. Overall, these `First use` instructions are now incompatible with `Manual Helm Installation` because the latter one relies on the password in `stackrox-admin-password.txt` file, not in `STACKROX_ADMIN_PASSWORD` environment variable.

Reply:
1. The script stores the password in `STACKROX_ADMIN_PASSWORD` instead of making a password file since it seemed redundant, it also prints the password at the end of the script so users can already copy-paste from there.
2. This is true, but I would rather have both installation methods work in the same way than provide two sets of instructions. Would you be okay with me changing the manual instructions to use `STACKROX_ADMIN_PASSWORD`as well?

Reply:
Re 1: My point is that the text

> 4. Log in as `admin` using the password in `STACKROX_ADMIN_PASSWORD`

is **ambiguous**. Perhaps only you, Piotr and I know what is `STACKROX_ADMIN_PASSWORD`.
Usually, when you mention environment variable you do it like

> 4. Log in as `admin` using the password in `$STACKROX_ADMIN_PASSWORD`

----

Re 2: I don't mind. Whichever way you like. Please make sure you thoroughly test manual instructions after the change.

Reply:
Re 1: I'm realizing that `STACKROX_ADMIN_PASSWORD` is not actually exported from the script (it just can't be). Therefore the user will not be able to run `echo $STACKROX_ADMIN_PASSWORD` after running the script.
You should somehow more accurately refer to the script's output for the password.
That said, if you teach the script to open the browser, this "First use" section may become unnecessary for quickstart and can be moved back to the manual instructions.